workspaceitems.md

```diff
@@ -0,0 +1,113 @@
+# WorkspaceItem Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/submission/workspaceitems**   
+
+Provide access to the workspaceitems. It returns the list of existent workspaceitems.
+
+Example: to be provided
+
+## Single Item
+**/api/submission/workspaceitems/<:id>**
+
+Provide detailed information about a specific workspaceitem. The JSON response document is as follow
+```json
+{
+  "id": 1911,
+  "lastModified": "2017-06-24T00:40:54.970+0000",
+  "sections": {
+  	 "collection": "05457c63-b392-4629-a373-f2d66ee9ee33",
+  	 "traditional-page1": {
+  	 	"dc.title" : [{value: "Sample Submission Item"}],
+  	 	"dc.contributor.author" : [
+  	 		{value: "Bollini, Andrea", authority: "rp00001", confidence: 600}
+  	 	]
+  	 },
+  	 "traditional-page2": {
+  	 	"dc.subject" : [{value: "Test"}, {value: "JSON"}, {value: "REST"}],
+  	 	"dc.description.abstract" : [
+  	 		{value: "This is a long multiline text\n This is a second line of the abstract"}
+  	 	]
+  	 },
+  	 "license": {
+  	 	acceptanceDate: "2017-06-24T00:40:54.970+0000",
+  	 	url: "http://dspace7.4science.it/api/core/bitstreams/8d33bdfb-e7ba-43e6-a93a-f445b7e8a1e2/content"
+  	 },
+  	 "uploads": [ 
+  	 	{
+  	 		metadata: {
+  	 			"dc.title" : [{value: "sample_file.pdf"}],
+  	 			"dc.description" : [{value: "Description of the sample file"}]
+  	 		},
+  	 		"sizeBytes": 8528,
+			"checkSum": {
+			    "checkSumAlgorithm": "MD5",
+			    "value": "9d8f0f9e369cf12159d47c146c499cf4"
+			},
+  	 		"url": "http://dspace7.4science.it/api/core/bitstreams/00001abf-b2e0-477a-99de-104db7cb6469/content",
+  	 		"accessConditions": [
+  	 			{
+	  	 			"type": "openaccess"
+  	 			},
+  	 			{
+	  	 			"type": "administrator"
+  	 			},
+  	 			{
+	  	 			"type": "embargo",
+	  	 			"group-uuid": "1faf7c51-2a14-4826-b0b1-f1c1d2d82dd7",
```

Comment:
I'm going to change group-uuid in groupUUID to be consistent. In the structural attributes we are following the camelCase convention. This is not the case for traditional-page1 that comes from the item-submission.xml